### ISSUE TITLE
Quest suggestion: Summit cross

### ISSUE THREAD

**Issue by FloEdelmann** [streetcomplete/StreetComplete#4095]
*Note: this is a follow-up issue to #561.*

### General
Affected tag(s) to be modified/added: [`summit:cross`](https://wiki.openstreetmap.org/wiki/Key:summit:cross)
Question asked: **Is there a summit cross here?**

### Checklist
Checklist for quest suggestions (see [guidelines](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md)):
- [x] 🚧 To be added tag is established and has a useful purpose:
  [3.2k uses](https://taginfo.openstreetmap.org/keys/summit%3Across); [may be useful for orientation](https://github.com/streetcomplete/StreetComplete/issues/561#issuecomment-325673300).
- [x] 🤔 Any answer the user can give must have an equivalent tagging (Quest should not reappear to other users when solved by one): `yes` or `no`
- [x] 🐿️ Easily answerable by everyone from the outside but a survey is necessary:
  It's easy to answer if the peak is clearly defined. Crosses are not easy to see on aerial imagery.
- [x] 💤 Not an overwhelming percentage of quests have the same answer (No spam):
  [93% of current uses](https://taginfo.openstreetmap.org/keys/summit%3Across#values) are `yes`, but that may be because armchair mappers only add it when it is present. From my experience, not every summit has a cross.
- [x] 🕓 Applies to a reasonable number of map data (Worth the effort):
  Hard to say because the filter will be quite complex. But this [simple filter](https://overpass-turbo.eu/s/1jbS) already shows at least 700 quests in a small area. Also, it [may be one of few quests in mountainous areas](https://github.com/streetcomplete/StreetComplete/issues/561#issuecomment-325575505).

@matkoniecz [prefers](https://github.com/streetcomplete/StreetComplete/issues/561#issuecomment-325673300) adding a node with `man_made = cross` instead (which would not be possible with StreetComplete), two people on the [discussion page](https://wiki.openstreetmap.org/wiki/Talk:Tag:summit:cross%3Dyes) had the same suggestion. I'd argue that the tags don't rule each other out, i.e. it's the same situation as with `bench = yes` on a bus stop with an `amenity = bench` nearby. Maybe tagging `summit:cross = yes` might even help armchair mappers to find the place to add a `man_made = cross` node.

### Ideas for implementation

**Element selection:** Similar to [`AddSummitRegister`](https://github.com/streetcomplete/StreetComplete/blob/master/app/src/main/java/de/westnordost/streetcomplete/quests/summit_register/AddSummitRegister.kt) after #4075:

```
nodes with
  natural = peak
  and name
  and (!summit:cross or summit:cross older today -16 years)
```

* if they are close to a hiking path
  (`ways with highway = path and sac_scale ~ mountain_hiking|demanding_mountain_hiking|alpine_hiking|demanding_alpine_hiking|difficult_alpine_hiking`)
* or if they are close to a hiking route
  (`relations with route = hiking`)
  → currently non-functional, as [routes are not fetched](https://github.com/streetcomplete/StreetComplete/issues/561#issuecomment-1141411959)
* or if they have `summit:register = yes`
* or if they have `summit:cross` (always resurvey)

Resurvey interval 16 years (vs. 4 years in `AddSummitRegister`) because constructing a summit cross is much more difficult than installing a summit register.

**Metadata needed:** Ask in same countries as `AddSummitRegister` (list gathered in https://github.com/streetcomplete/StreetComplete/issues/561#issuecomment-325623974)

**Proposed UI:** Yes/No

**Comment by matkoniecz** [comment]
If `summit:cross = yes` and `man_made = cross` can coexist, then from

> @matkoniecz prefers adding a node with `man_made = cross` instead

instead can be dropped

**Comment by matkoniecz** [comment]
> [93% of current uses](https://taginfo.openstreetmap.org/keys/summit%3Across#values) are yes, but that may be because armchair mappers only add it when it is present

Definitely that is because mappers only add it when it is present (and not only armchair ones). If anything, I would worry about `no` spam
